pipeline/luigi/CompileVCFFiles.py

```diff
@@ -1447,6 +1447,60 @@ def run(self):
 
 
 @requires(RunDiffAndAppendChangeTypesToOutput)
+class RunDiffAndAppendChangeTypesToOutputReports(luigi.Task):
+    def _extract_release_date(self, version_json):
+        with open(version_json, 'r') as f:
+            j = json.load(f)
+            return datetime.datetime.strptime(j['date'], '%Y-%m-%d')
+
+
+    def _extract_file(self, archive_path, tmp_dir, file_path):
```

Comment:
This function and the above could probably be easily refactored out, as it is also used in `RunDiffAndAppendChangeTypesToOutput`